.circleci/config.yml

```diff
@@ -130,7 +130,7 @@ workflows:
               # of Python, we probably have to do something here.
               pythonVersion:
                 - "3.9"
-                - "3.11"
+                - "3.12"
```

Comment:
I am confused.  Aren't all these changes already in `master`?

Reply:
The "sych with Upstream" commit smashes a bunch together. This is not the way to re-sync (so this PR will be un-mergable). We can discuss in N&B tomorrow.

The tl;dr is: "git merge master" to resync a branch.

Reply:
This PR is building on top of the PR #1381 I use to poke around with CI?
That is a misunderstanding, I do not intend to merge that branch - Anything good in there I have created small new PRs for.  I really only use it as a sandbox.  Please don't build on top of it.

Reply:
> The "sych with Upstream" commit smashes a bunch together. This is not the way to re-sync

I didn't mean to merge master in, I intended to throw away the prior work in that PR to start over again with what was on master at that time.

Reply:
Sorry, I will close this PR.